lib/proxy.js

```diff
@@ -699,7 +741,7 @@ function redirectToHttps(req, res, target, ssl, log) {
 
 function startsWith(input, str) {
   return input.slice(0, str.length) === str &&
-    (input.length === str.length || input[str.length] === '/')
+    (input.length === str.length || (input[str.length] === '/' || input[str.length] === '?'))
```

Comment:
what issue is this change fixing?

Reply:
ex function startWith
input | str | input[str.length]
/x/services/xxx?wsdl /x/services/xxx

Reply:
Yes, I understand that, but what issue does i fix?

Reply:
When I try to get a soap WSDL with "?wsdl", I get {"name":"redbird","hostname":"localhost.localdomain","pid":2820,"level":40,"src":"127.0.0.1","url":"/connecta-public-services/services/WSReservationService&wsdl","msg":"no valid route found for given source","time":"2018-02-02T11:10:14.182Z","v":0}^

but same request without "?wsdl" is working